index.html

```diff
@@ -4419,57 +4294,31 @@ <h3>Parsing content</h3>
     </section>
   </section>
 
-  <section class="informative"> <h2>Security mechanisms</h2>
+  <section class="informative"> <h2>Security mechanisms for implementations</h2>
     <p>
       In order to support security policies concerning Web NFC, the following
-      mechanisms are recommended for the implementations.
+      mechanisms are recommended for implementations.
     </p>
     <section> <h3><dfn>Obtaining permission</dfn></h3>
-      Implementations SHOULD use one or more of the following mechanisms to
-      <dfn>obtain permission</dfn>.
-      <ul>
-        <li>
-          A mechanism for obtaining <a>expressed permission</a> of the user.
-        </li>
-        <li>
-          A mechanism for <a>ask for forgiveness</a> policies.
-        </li>
-      </ul>
-      <section> <h4><dfn>Expressed permission</dfn></h4>
-        <p>
-          Denotes an explicit permission given by the user.
-          The [[[PERMISSIONS]]] API is suggested to be used by
-          UAs for implementing NFC related permissions in order to minimize
-          the need for user prompting.
-        </p>
-        <p>
-          The
-          <a href="https://www.w3.org/TR/permissions/#dictdef-permissiondescriptor">
-          <dfn>Web NFC permission name</dfn></a> is
-          <a href="https://github.com/w3c/permissions/issues/47">defined</a> as
-          "`nfc`".
-        </p>
-      </section>
-      <section> <h4><dfn>Ask for forgiveness</dfn></h4>
-        <p>
-          Denotes an unobtrusive way to notify the user about an ongoing operation
-          with a possible action to cancel or undo the operation, and to
-          handle future occurences of the same operation by the same action.
-        </p>
-      </section>
+      <p>
+        Implementations SHOULD use one or more of the following mechanisms to
```

Comment:
"one or more" does't make sens anymore since there's only one right?

Reply:
Thanks for noticing. Fixed.